README.md

```diff
@@ -109,25 +109,50 @@ That's it! You now have a fully-functional The Spaghetti Detective server that y
 
 # Configure The Spaghetti Detective OctoPrint Plugin to use your own server
 
-Before you can configure The Spaghetti Detective OctoPrint Plugin to use your own server, you need add a printer to The Spaghetti Detective server you just built and obtain the secret token for that. To do so:
+Before you can configure The Spaghetti Detective OctoPrint Plugin to use your own server, you need add a printer to The Spaghetti Detective server you just built and obtain the 6-digit Verification Code for that. To do so:
 
 1. Pointing your browser to `http://your_server_ip:3334`.
 
-1. Log in as a user (you can just login with `root@example.com` but it's more secure to use a non-admin user).
+2. Log in as a user (you can just login with `root@example.com` but it's more secure to use a non-admin user).
 
-1. Add a new printer as described in [this guide](https://www.thespaghettidetective.com/docs/octoprint-plugin-setup/#step-2-add-a-new-printer) and obtain the secret token. *Note: Do it on your own server, not on [The Spaghetti Detective cloud](https://app.thespaghettidetective.com).*
+3. Add a new printer as described in [this guide](https://www.thespaghettidetective.com/docs/octoprint-plugin-setup/#step-2-add-a-new-printer) and obtain the 6-digit Verification Code. *Note: Do it on your own server, not on [The Spaghetti Detective cloud](https://app.thespaghettidetective.com).*
 
-Then, on The Spaghetti Detective plugin settings page:
+Then, navigate to octoprint to setup the plugin side of things:
 
-1. Check the box "I have my own TSD server. Don't check this unless you know what you are doing."
+1. Make sure that you have installed the [TSD plugin](https://www.thespaghettidetective.com/docs/octoprint-plugin-setup/#step-1-install-access-anywhere---the-spaghetti-detective-plugin-in-octoprint).
 
-2. Enter `http://your_server_ip:3334`.
+2. After restarting, close the wizard, there is something that needs to be configured first.
 
-3. Enter the secret token you copied from the previous step.
+After you have closed the wizard or if you already had the TSD plguin installed:
+
+1. Open the OctoPrint Settings and navigate to "Access Anywhere - The Spaghetti Detective"
+
+2. Click "Settings"
+
+3. Find and change the Server Address to `http://your_server_ip:3334/` (use https:// if you have it, if you aren't sure, just use http://) you MUST include the "http://" and trailing "/"
 
 4. Click "Save". OctoPrint isn't necessarily smarter than Django after all.
 
-![Site configuration](https://raw.githubusercontent.com/TheSpaghettiDetective/TheSpaghettiDetective/master/docs/plugin_config.png)
+<img src="https://github.com/ryryog25/TSD-fork-resources/blob/main/Screen%20Shot%202021-03-02%20at%209.20.58%20AM.png?raw=true" width="500">
```

Comment:
Can you add the images as part of your PR? It's usually a good idea to include everything needed to be in the same repo to make it self-contained.

Reply:
Sure!